### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by cmnord**
Everything is prettier!

### CODE REVIEW

**Comment by cmnord** [comment]
Oh also I might have forgotten to put title blocks on some pages.

**Comment by cmnord** [comment]
Also the script for expanding text should probably be in a separate file (thanks @stefren)

File: static/style.css [hunk 1]
```diff
@@ -3,28 +3,142 @@ html {
...
-    margin: 0.5rem 0;
+    border: 0px;
+    border-radius: 0px;
+    font-size: 14px;
+    margin-bottom: 1vh;
+    font-family: "Courier New",Courier,monospace;
+    padding: 5px;
+} .form-control:focus {
+    -webkit-box-shadow: 0px 0px 5px 2px #0b3b4b;
+    -moz-box-shadow: 0px 0px 5px 2px #0b3b4b;
```
Comment (anishathalye) [inline static/style.css]: don't need those vendor prefixes, see http://caniuse.com/#search=box-shadow

**Comment by anishathalye** [comment]
Awesome, looks really nice!

I really like the overall styling, and I also like how the voting view is color-coded now!

I left some comments inline.

Some other comments:
* We're using Montserrat and Lato, but those fonts aren't included on most systems. To account for this, we can include something like [this](https://github.com/techx/hackmit-splash/blob/f5667c19de6f86fe206529a1911463aeb4d9e06f/index.html#L12) in `layout.html`.
* While the Doge is cute (and [public domain](https://pixabay.com/en/shiba-inu-dog-shiba-inu-japanese-944510/), apparently), it might be nicer to just design our own simple [placeholder logo].(http://placehold.it/). But rather than having a logo, would it be nicer just to let people specify their own branding text that can be set via the `config.yaml` (so it'll be less effort for people to customize)?
* At some point, it would be nice to extract the brand colors from the CSS and figure out how we can specify them in the `config.yaml` (so others can easily customize - see #13), though this doesn't have to be done in _this_ PR (also, I'm not sure exactly how this should be done).
* Table in admin view resizes when you click on the "see more" arrow. Given that it's the admin page, it's probably fine to just not bother with shortening the text.
* The add project / add judge things submit when you hit enter, and that makes it hard to manually enter more than one thing at once (you need to copy and paste from a text editor) cause you can't add newlines.
* I haven't tried it yet, but it seems like it might be easy to accidentally hit the skip button - can we add some spacing in between the [Previous | Current] and [Skip]?
* For specifying sizes of things, it's nice to be consistent with units. One good way to do it is to specify a `font-size` on `html`, using a media query to set it appropriately for different screen sizes, and then use `rem` units for everything else, so it scales according to the font size (`rem` = proportional to Root element EM size). See [here](http://www.anishathalye.com/static/css/main.css) for an example.

File: static/style.css (hunk 1 above)
Comment (cmnord) [inline static/style.css]: fixed 3450f6f5e8152f0099688113c395e059c63d045c

**Comment by cmnord** [comment]
- Fonts and placeholder image fixed in 3450f6f5e8152f0099688113c395e059c63d045c.

- Are you sure that you don't want shortened text in the table in admin view? It makes it inconvenience to read the whole thing (unless there's an "expand all" button), but I feel like that's not really necessary.
- Skip button has a popup now 94825586ffcb5e224389ab66160eb88de0f6da58 -- this is a good enough solution to accidental presses I think

TODO:
- Fix delete button
- Change input back to textarea
- Consistent sizing
- Easy customization

**Comment by anishathalye** [comment]
Btw, a couple minor changes have been made in the frontend in some recent commits. To see what has changed, you can run `git diff b9559fb e0c711a -- :/templates`.

**Comment by anishathalye** [comment]
Even more stuff changed... sorry about that (`master` is moving pretty quickly 😛)

Do a `git fetch && git diff b9559fb origin/master -- :/templates :/gavel/templates` to see what has changed.

It's pretty minor: `9 files changed, 56 insertions(+), 2 deletions(-)`. Luckily, there aren't any significant changes to the frontend, mostly just shuffling stuff around, so it should be straightforward to rebase onto the tip of the tree.

**Comment by anishathalye** [comment]
Superseded by #29

**Pull request closed without merging**